native/src/main/java/io/ballerina/stdlib/grpc/builder/syntaxtree/utils/ClientSampleSyntaxTreeUtils.java

```diff
@@ -0,0 +1,430 @@
+        ExpressionNode node = getCheckExpressionNode(getRemoteMethodCallActionNode(
+                getBuiltinSimpleNameReferenceNode(CONST_ENDPOINT), method.getMethodName(),
+                method.getInputType() != null ? CONST_REQUEST : ""));
+        VariableDeclaration streamingCallVariable = new VariableDeclaration(bindingPatternNode, node);
+        return streamingCallVariable.getVariableDeclarationNode();
+    }
+
+    private static ExpressionStatementNode getStreamSendValueStatementNode(Method method) {
+        ExpressionNode node = getCheckExpressionNode(getRemoteMethodCallActionNode(
+                getBuiltinSimpleNameReferenceNode(CONST_STREAMING_CLIENT),
+                "send" + getMethodType(method.getInputType()),
+                method.getInputType() != null ? CONST_REQUEST : ""));
+        return getCallStatementNode(node);
+    }
+
+    private static ExpressionStatementNode getStreamCompleteStatementNode() {
+        ExpressionNode node = getCheckExpressionNode(getRemoteMethodCallActionNode(
+                getBuiltinSimpleNameReferenceNode(CONST_STREAMING_CLIENT), "complete"));
+        return getCallStatementNode(node);
+    }
+
+    private static VariableDeclarationNode getStreamReceiveValueDeclarationNode(Method method, String filename) {
+        TypedBindingPatternNode bindingPatternNode = getTypedBindingPatternNode(
+                getSimpleNameReferenceNode(method.getOutputPackageType(filename) + method.getOutputType() + "? "),
+                getCaptureBindingPatternNode(CONST_RESPONSE));
+        ExpressionNode checkExpressionNode = getCheckExpressionNode(getRemoteMethodCallActionNode(
+                getSimpleNameReferenceNode(CONST_STREAMING_CLIENT), "receive" + getMethodType(method.getOutputType())));
+        VariableDeclaration streamingCallVariable = new VariableDeclaration(bindingPatternNode, checkExpressionNode);
+        return streamingCallVariable.getVariableDeclarationNode();
+    }
+
+    private static VariableDeclarationNode getInputDeclarationStatement(Method method, String filename,
+                                                                        Map<String, Message> msgMap) {
+        TypedBindingPatternNode bindingPatternNode = getTypedBindingPatternNode(
+                NodeFactory.createSimpleNameReferenceNode(AbstractNodeFactory.createIdentifierToken(
+                        method.getInputPackagePrefix(filename) + method.getInputType() + " ")),
+                getCaptureBindingPatternNode(CONST_REQUEST));
+        ExpressionNode node = null;
+        switch (method.getInputType()) {
+            case "int":
+            case "float":
+            case "decimal":
+                node = getNumericLiteralNode(1);
+                break;
+            case "boolean":
+                node = getBooleanLiteralNode(true);
+                break;
+            case "string":
+                node = getStringLiteralNode("Hello");
```

Comment:
```suggestion
                node = getStringLiteralNode("xyz");
```

Reply:
Updated in https://github.com/ballerina-platform/module-ballerina-grpc/pull/952/commits/10fa6fdebc07f49e1c8cf191abb3d8b7515638ab